lib/screenplay_web/auth_manager/auth_manager.ex

```diff
@@ -24,21 +26,26 @@ defmodule ScreenplayWeb.AuthManager do
 
   def resource_from_claims(_), do: {:error, :invalid_claims}
 
-  @spec claims_access_level(Guardian.Token.claims()) :: access_level()
+  @spec claims_access_level(Guardian.Token.claims()) :: list(access_level())
   def claims_access_level(%{"roles" => roles}) when not is_nil(roles) do
-    cond do
-      @screenplay_admin_role in roles ->
-        :emergency_admin
-
-      @screens_admin in roles ->
-        :screens_config_admmin
-
-      true ->
-        :read_only
+    access_levels =
+      []
+      |> append_if(@screenplay_emergency_admin_role in roles, :emergency_admin)
+      |> append_if(@screens_admin in roles, :screens_admin)
+      |> append_if(@pa_message_admin in roles, :pa_message_admin)
+
+    if access_levels == [] do
+      [:read_only]
```

Comment:
Is there a reason you don't also have `read_only` if you have some higher level of access?

Reply:
It might actually be something we can remove and have the function return an empty list instead. The `:read_only` permission is not used directly because we enforce auth for all endpoints. If you make it through that, read only is implied.